docker-compose.yml

```diff
@@ -1,9 +1,26 @@
-spool:
-  build: .
-  volumes:
-    - ./spool:/usr/src/app/spool
-    - ./tests:/usr/src/app/tests
-    - ./pytest.ini:/usr/src/app/pytest.ini
-    - ./setup.cfg:/usr/src/app/setup.cfg
-    - ./setup.py:/usr/src/app/setup.py
-  command: py.test --verbose
+version: '2'
+
+services:
+  spool:
+    build: .
+    volumes:
+      - ./docs:/usr/src/app/docs
```

Comment:
If one wishes to modify the docs, and/or build the docs, and have the changes synced on the host and in the container, yes. We could probably simply mount the whole thing:

```yaml
- .:/usr/src/app
```